docs/ops/activation/Swish_4.md

```diff
@@ -0,0 +1,54 @@
+## Swish <a name="Swish"></a>
+
+**Versioned name**: *Swish-4*
+
+**Category**: *Activation*
+
+**Short description**: Swish takes one input tensor and produces output tensor where the swish function is applied to the tensor elementwise.
+
+**Detailed description**: [Reference.](https://arxiv.org/pdf/1710.05941.pdf)    
```

Comment:
Move `[Reference.](https://arxiv.org/pdf/1710.05941.pdf) ` to the end of the paragraph and replace the word "Reference" to some more wordy text like 'The [article](https://arxiv.org/pdf/1710.05941.pdf) where the Swish operation is introduced.'

Reply:
Done.